package.json

```diff
@@ -173,7 +175,9 @@
     "**/@types/jest-environment-puppeteer/@jest/types": ">=29 <30",
     "**/@types/jest-environment-puppeteer/jest-environment-node": ">=29 <30",
     "**/eslint-config-next/eslint-plugin-react-hooks": "^4.3.0",
-    "**/ng-packagr/rollup": "^3.9.1",
-    "**/workbox-build/rollup": "^3.9.1"
+    "**/eslint-config-react-app/eslint-plugin-jest": "^26.9.0",
+    "**/vite/rollup": "^3.10.0",
```

Comment:
Why do we need those dependencies as forced resolutions?

Reply:
To prevent having multiple versions of rollup in the project where it isn't clear which one is used and warnings about unsupported `--bundleConfigAsCjs` flag if the wrong version is used.